docs/integrations/pydantic.md

```diff
@@ -30,23 +30,38 @@ We can create a Strawberry type by using the
 
 ```python
 import strawberry
+from strawberry.experimental.pydantic import auto
```

Comment:
considered we will use this for django, should we move it somewhere else? 🤔

Reply:
Yeah I'm happy to move it, just wasn't sure where it belonged. Ideally I think we'd expose it from the top level namespace for ease of use, but where the code should go I'm not sure